src/credentials/CertificationDeclaration.h

```diff
@@ -62,6 +63,38 @@ struct CertificationElements
     bool DACOriginVIDandPIDPresent;
 };
 
+struct CertificationDeclarationContent
+{
+    uint16_t formatVersion;
+    uint16_t vendorId;
+    bool containsPID;
```

Comment:
This field is redundant. PID is a mandatory field in the CD struct and should always be present. If PID field is not present the decoding of CD struct fails.

Reply:
Addressed in b5f45aa